distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Delimited_Reader.enso

```diff
@@ -48,6 +49,10 @@ read_file format file on_problems:Problem_Behavior =
        reached and restart parsing with an increased limit.
     result = file.with_input_stream [File_Access.Read] stream->
         read_stream format stream on_problems related_file=file
+
+    if result.is_error.not then
+        Telemetry.log "Delimited_Reader" "Read CSV file: column_count={}, row_count={}" [result.column_count, result.row_count]
```

Comment:
OK, this looks as [specified](https://github.com/enso-org/enso/issues/12464#issuecomment-2713545076). Just instead in Java it is written in Enso. Good.